docs/io/ascii/write.rst

```diff
@@ -338,6 +338,8 @@ Starting with ``astropy`` 2.0 it is possible to store not only standard
 - `astropy.coordinates.Distance`
 - `astropy.coordinates.EarthLocation`
 - `astropy.coordinates.SkyCoord`
+- `astropy.table.NdarrayMixin`
+- Coordinate representation types such as `astropy.coordinates.SphericalRepresentation`
```

Comment:
It would be great to have an example in the docs of putting a Representation in a Table.
Also, when did general CoordinateFrame objects become allowed in Tables?!? I thought it was only SkyCoord.